styles.css/main.css

```diff
@@ -111,26 +99,36 @@ h1, h2, h3, h4, h5, h6 {font-family:rubik;}
 
 }
 
-.tab2{
+.bio table {
```

Comment:
Again, don't use wrapper divs if it's not necessary.  Just use more specific selectors